packages/anoncreds/src/protocols/proofs/v1/__tests__/v1-indy-proof-proposal.e2e.test.ts

```diff
@@ -103,4 +111,114 @@ describe('Present Proof', () => {
       protocolVersion: 'v1',
     })
   })
+
+  test('Alice Creates oob proof proposal for Faber', async () => {
+    await issueLegacyAnonCredsCredential({
+      issuerAgent: faberAgent,
+      issuerReplay: faberReplay,
+      holderAgent: aliceAgent,
+      holderReplay: aliceReplay,
+      issuerHolderConnectionId: faberConnectionId,
+      offer: {
+        credentialDefinitionId,
+        attributes: [
+          {
+            name: 'name',
+            value: 'Alice',
+          },
+          {
+            name: 'age',
+            value: '99',
+          },
+        ],
+      },
+    })
+    testLogger.test('Alice creates oob proof proposal for Faber')
+    const { message } = await aliceAgent.proofs.createProofProposal({
+      protocolVersion: 'v1',
+      proofFormats: {
+        indy: {
+          name: 'ProofRequest',
+          version: '1.0',
+          attributes: [
+            {
+              name: 'name',
+              value: 'John',
+              credentialDefinitionId,
+              referent: '0',
+            },
+          ],
+          predicates: [
+            {
+              name: 'age',
+              predicate: '>=',
+              threshold: 50,
+              credentialDefinitionId,
+            },
+          ],
+        },
+      },
+      comment: 'V1 propose proof test',
+    })
+    const { outOfBandInvitation } = await aliceAgent.oob.createInvitation({
+      messages: [message],
+      autoAcceptConnection: true,
+    })
+    await faberAgent.oob.receiveInvitation(outOfBandInvitation)
+    testLogger.test('Faber waits for proof proposal message from Alice')
+    let faberProofExchangeRecord = await waitForProofExchangeRecord(faberAgent, {
```

Comment:
Can you use waitForProofExchangeRecordSubject? This method can have timing issues